src/components/dropDownLists/SSIDropDownListItem/index.tsx

```diff
@@ -45,6 +46,8 @@ const SSIDropDownListItem: FC<IProps> = (props: IProps): JSX.Element => {
         return <SSIDownloadIcon color={fontColor} />;
       case HeaderMenuIconsEnum.SETTINGS:
         return <SettingsIcon color={fontColor} />;
+      case HeaderMenuIconsEnum.QR:
+        return <SSIQRIcon primaryColor={fontColor.toString()} secondaryColor={fontColor.toString()} />;
```

Comment:
why is there a toString used if other icons do not need this? please keep consistency

Reply:
i have made an update to this by changing the prop types of the `SSIQRIcon` component to use `ColorValue` from react-native instead of `string`. Since i noticed this is what the other icon components are doing. So i have updated the usage of this to use `fontColor` directly.

I believe the reason why there are primary and secondary colors on this component already is because it supports gradient colors.